### PR TITLE
 Fix casing typos in NuGet and MSBuild

### PR DESCRIPTION
**Pull request by dannyvv**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5358)

### CODE REVIEW

**Comment by dannyvv** [comment]
@NickGerleman: We have typos in `changelog.json` and `changelog.md` too. What is our policy of fixing typo's there? Should I fix those too, or leave as-is?

**Comment by dannyvv** [comment]
This change stems form feedback in PR #5283 where I made a few typos in the name and discovered the same casing mistakes were all over the place.

**Comment by NickGerleman** [comment]
> @NickGerleman: We have typos in `changelog.json` and `changelog.md` too. What is our policy of fixing typo's there? Should I fix those too, or leave as-is?

My preference would be to keep them verbatim with what engineers added

File: vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs [hunk 1]
```diff
@@ -37,58 +37,58 @@ public class TestClass
...
+
+            // A custom target in the test project ensures the x64 versions of these assemblies are built
+            // before the unittest is run.
+            // I could not find a way for the dotnet core project to NOT be "Any CPU" and then produce x64 binaries
+            // in that configuration. I tried all the usual tricks for the netcore project to match the universal
+            // multi platform support.
+            references.Add($@"{targetRoot}\X64\{configuration}\Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed.dll");
+            references.Add($@"{targetRoot}\X64\{configuration}\Microsoft.ReactNative\Microsoft.ReactNative.winmd");
+
+            // Add the windows 10 sdk assemblies for UWP. We have a compile time safeguard in place to
```
Comment (asklar) [inline vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs:79]: Windows
SDK

File: vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs [hunk 2]
```diff
@@ -37,58 +37,58 @@ public class TestClass
...
+            // ensure that it matches the version that is defined in our MSBuild projects matches the one
+            // used in our unittests. Unfortunately C# doesn't allow constants to flow into code, nor do any
+            // of the unitest runners are allowed to pick up a variable from the build file. And given the many
+            // ways one can run inttests, this seemed to be the most reasonable out of a lot of poor options.
```
Comment (asklar) [inline vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs:83]: integration tests?

File: vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs [hunk 3]
```diff
@@ -13,163 +13,163 @@
...
     {
-      m_cancellationToken = cancellationToken;
-    }
+        private readonly CancellationToken m_cancellationToken;
 
-    /// <summary>
-    /// The main entry point to run the code generation based on the given options.
-    ///
-    /// Main phases of execution are:
-    /// 1. Load the code into roslyn
```
Comment (asklar) [inline vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs:29]: Roslyn

File: vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs [hunk 4]
```diff
@@ -13,163 +13,163 @@
...
-    private async Task<bool> TryWriteFileIfDifferent(string path, string contents)
-    {
-      try
-      {
-        if (File.Exists(path))
+        /// <summary>
+        /// The main entry point to run the code generation based on the given options.
+        ///
+        /// Main phases of execution are:
+        /// 1. Load the code into roslyn
```
Comment (asklar) [inline vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs:29]: Roslyn

File: vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs (hunk 1 above)
Comment (dannyvv) [reply vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs:79]: I'll send another PR with these typo fixes.

Comment (dannyvv) [reply vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs:79]: #5366

File: vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs (hunk 2 above)
Comment (dannyvv) [reply vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs:83]: #5366

File: vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs (hunk 4 above)
Comment (dannyvv) [reply vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs:29]: #5366

File: vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs (hunk 3 above)
Comment (dannyvv) [reply vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs:29]: #5366

**Pull request merged**